roles/certificate_bundle/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: 'Generate Certs'
+  command: foreman-proxy-certs-generate --foreman-proxy-fqdn {{ certificate_bundle_fqdn }} --certs-tar /root/{{ certificate_bundle_fqdn }}.tar.gz {{ certificate_bundle_options }} # noqa 204
```

Comment:
I think I'd prefer to allow the user to configure where the tarball lands.

so something like

```
certificate_bundle_tar: "/root/{{ certificate_bundle_fqdn }}.tar.gz"
```

in defaults and then using it here?